Resources/Prototypes/Corvax/Entities/Objects/Misc/books.yml

```
@@ -0,0 +1,1324 @@
+- type: entity
+  parent: BookBase
+  id: BookExpensiveCrystal
+  name: Кристаллы особой ценности
+  description: Данная книга рассказывает о материалах, которые в обиходе было принято называть Блюспейс и Редспейс кристаллами.
+  components:
+    - type: Sprite
+      sprite: Corvax/Objects/Misc/books.rsi
+      layers:
+        - state: book_expensive_crystal
```

я бы переделал это под конструктор слоёв как в остальных книгах, но если не хочешь то тут нужно сделать просто
```suggestion
      state: book_expensive_crystal
```

---

Там все спрайты авторские и сделаны именно под тематику книги, так што лучше не менять.

---

Ща всё поправлю